<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>411</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Monitor de Latencia</string>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Chile"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_device">
     <attribute name="title">
      <string>Dispositivo</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_connection">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>381</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Conexión</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_connection" columnminimumwidth="100,0,0">
        <item row="0" column="0">
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>Puerto:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_baudrate">
          <property name="text">
           <string>Baudrate:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_scan">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QComboBox" name="comboBox_port">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_baudrate">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>115200</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
          <property name="clearButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_connect">
          <property name="text">
           <string>Conectar</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>381</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Información</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_info">
        <item row="0" column="0">
         <widget class="QLabel" name="label_id_left">
          <property name="text">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_id_right">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>No disponible</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_version_left">
          <property name="text">
           <string>Versión</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_version_right">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>No disponible</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_precision_left">
          <property name="text">
           <string>Precisión</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_precision_right">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>No disponible</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_mac_left">
          <property name="text">
           <string>Dirección MAC</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_mac_right">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>No disponible</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_config">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>381</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Configuración</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>107</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_config">
        <item row="0" column="0">
         <widget class="QLabel" name="label_packetSize">
          <property name="text">
           <string>Tamaño de Paquete [bytes]</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_packetSize">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>1500</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>512</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_packetInterval">
          <property name="text">
           <string>Intervalo de envío [ms]</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_packetInterval">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_packetDst">
          <property name="text">
           <string>Destino</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_dst">
          <item>
           <widget class="QCheckBox" name="checkBox_broadcast">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Broadcast</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_dst">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_content">
          <property name="text">
           <string>Contenido</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_content">
          <item>
           <widget class="QCheckBox" name="checkBox_content">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Personalizado</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_content">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cargar</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_latency">
     <attribute name="title">
      <string>Latencia</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_chart_latency">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>381</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Gráfico</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>361</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_chart">
        <item>
         <widget class="QComboBox" name="comboBox_chartType"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>371</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>Registro</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_log">
          <property name="text">
           <string>Comenzar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_dataCount_left">
          <property name="text">
           <string>Mediciones</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_dataCount_right">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ancho de banda</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_bandwidth">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Tasas de transmisión</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
        <item row="0" column="0">
         <widget class="QLabel" name="label_bandwidth_in_top">
          <property name="text">
           <string>Entrada</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_bandwidth_in_bottom">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>No disponible</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_bandwidth_out_top">
          <property name="text">
           <string>Salida</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_bandwidth_out_bottom">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>No disponible</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
